modules/rosa-classic-aws-rhcos-ami.adoc

```diff
@@ -0,0 +1,302 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa-aws-amis.adoc
+
+[id="rosa-classic-aws-rhcos-ami_{context}"]
+= ROSA AMIs for {product-title}
+
+Red Hat provides RHCOS AMIs that are valid for the supported AWS regions and instance types that you can manually specify for your {product-title} nodes.
+
+List of Regions:
```

Comment:
I think remove the list of regions here.

Reply:
Sorry, that was my verification list to make sure I didn't lose any regions when converting; they will be removed before publishing.